src/codegen/sys/tests.rs

```diff
@@ -145,7 +145,11 @@ fn prepare_cconsts(env: &Env) -> Vec<CConstant> {
                     // see the GValue machinery around them for example
                     constants.push(CConstant {
                         name: format!("(guint) {}", member.c_identifier),
-                        value: member.value.clone(),
+                        value: member
+                            .value
+                            .parse::<i32>()
```

Comment:
Are you sure these can never be an actual `u32` that is bigger than `i32::MAX`?

Reply:
The `unwrap_or_else` should cover that?

Reply:
True